unified-runtime/test/conformance/device/urDeviceGetInfo.cpp

```diff
@@ -1758,7 +1758,7 @@ TEST_P(urDeviceGetInfoTest, SuccessAtomicFenceScopeCapabilities) {
   ASSERT_EQ(property_value & UR_MEMORY_SCOPE_CAPABILITY_FLAGS_MASK, 0);
 }
 
-TEST_P(urDeviceGetInfoTest, SuccessBFloat64) {
+TEST_P(urDeviceGetInfoTest, SuccessBFloat16) {
   size_t property_size = 0;
   const ur_device_info_t property_name = UR_DEVICE_INFO_BFLOAT16;
```

Comment:
this is for the old deprecated bfloat16 query, there's a pr open to remove it https://github.com/intel/llvm/pull/17053/files#diff-4ad6c68f3c3e27ec6c124dab1d1c2cd28dc6bba6f4e0960f44eec79b3c0c1c98

for testing if you could just copy this function but rename the test and update property_name that'd be ideal, like this:
```
TEST_P(urDeviceGetInfoTest, SuccessBFloat16Conversions) {
  size_t property_size = 0;
  const ur_device_info_t property_name = UR_DEVICE_INFO_BFLOAT16_CONVERSIONS_INTEL;
```